app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+    <string name="cancel">Hủy</string>
+    <string name="are_you_sure_you_want_to_cancel">Bạn có chắc chắn muốn hủy?</string>
+    <string name="edit">Chỉnh sửa</string>
+    <string name="avatar_shop">Ảnh cửa hàng</string>
+    <string name="create_new_shop">Tạo cửa hàng</string>
+    <string name="open_forever">Mở cửa hằng ngày:</string>
+    <string name="auto_reject_time">Thời gian tự động từ chối: </string>
+    <string name="cover_image">Ảnh bìa</string>
+    <string name="two_dot">:</string>
+    <string name="choose_image">Chọn ảnh</string>
+    <string name="create">Tạo</string>
+    <string name="open_time">Thời gian mở cửa :</string>
+    <string name="open_daily">Mở cửa hằng ngày :</string>
+    <string name="avatar">Ảnh đại diện : </string>
+    <string name="time_automatically_refused_order">Thời gian tự động từ chối đơn đặt hàng :</string>
+    <string name="choose_file">Chọn tệp</string>
+    <string name="edit_shop">Chỉnh sửa cửa hàng</string>
+    <string name="choose_time">Chọn thời gian</string>
+    <string name="time_auto_close">Thời gian tự động đóng cửa :</string>
+    <string name="save">Lưu</string>
+    <string name="zero">0</string>
+    <string name="name_is_empty">Tên không được để trống</string>
+    <string name="description_is_empty">Mô tả không được để trống</string>
+    <string name="create_shop_successful">Tạo cửa hàng thành công!</string>
+    <string name="please_wait">Làm ơn chờ..</string>
+    <string name="loading">Đang tải…</string>
+    <string name="you_must_choose_image">Bạn cần phải chọn ảnh!</string>
+    <string name="blocked">Đã bị chặn</string>
+    <string name="wait">Chờ</string>
+    <string name="active">active</string>
+    <string name="inactive">inactive</string>
+    <string name="create_new_product">Tạo sản phẩm</string>
+    <string name="product_name">Tên sản phẩm</string>
+    <string name="category">Thể loại</string>
+    <string name="start_hour">Giờ bắt đầu</string>
+    <string name="end_hour">Giờ kết thúc</string>
+    <string name="price">Giá</string>
+    <string name="update_success">CẬp nhật thành công</string>
+    <string name="top_shop">Cửa hàng nổi bật</string>
+    <string name="categories">Thể loại</string>
+    <string name="top_product">Sản phẩm nổi bật</string>
+    <string name="create_product_successful">Tạo sản phẩm thành công!</string>
+    <string name="edit_product">Chỉnh sửa sản phẩm</string>
+    <string name="update_successful">Cập nhật thành công</string>
+    <string name="note">Note :</string>
+    <string name="order_successful">Đặt hàng thành công!</string>
+    <string name="accepted_status">accepted</string>
+    <string name="rejected_status">rejected</string>
+    <string name="input_your_note">Nhập ghi chú</string>
+    <string name="add_note">Thêm ghi chú</string>
+    <string name="reject_all">Từ chối tất cả</string>
+    <string name="accept_all">Chấp nhận tất cả</string>
+    <string name="leave">Rời khỏi</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.leave` appears to be unused
 Unused resources make applications larger and slow down builds.